parquet-hadoop/src/test/java/org/apache/parquet/hadoop/TestInputFormat.java

```diff
@@ -398,7 +399,10 @@ public void testGetFootersReturnsInPredictableOrder() throws IOException {
       if (i > 0) {
         url.append(',');
       }
-      url.append("file:").append(file.getAbsolutePath());
+      if (!SystemUtils.IS_OS_WINDOWS) {
+        url.append("file:");
+      }
+      url.append(file.getAbsolutePath());
```

Comment:
I don't have any experience running Hadoop on Windows but I am not sure this is the good approach to pass the tests. I think it would be better to expect the same configuration on the different systems and handle OS relevant paths in the production code if required.

Reply:
I've changed this test to use URIs instead. Now it should work platform independent.